Common/include/geometry/primal_grid/CPrimalGridFEM.hpp

```diff
@@ -213,8 +213,16 @@ class CPrimalGridFEM final: public CPrimalGrid {
    */
   void GetCornerPointsAllFaces(unsigned short &numFaces,
                                unsigned short nPointsPerFace[],
-                               unsigned long  faceConn[6][4]) override;
+                               unsigned long  faceConn[6][4]) const override;
+private:
+  /*!
+   * \brief Get the number of faces of the element.
+   * \param[in] elementType - element type
+   * \return number of faces
+   */
+  static unsigned short elementtype_to_nFaces(unsigned short elementType);
```

Comment:
```suggestion
  static inline unsigned short nFacesOfElementType(unsigned short elementType) {
    switch (elementType) {
      case TRIANGLE: return 3;
      case QUADRILATERAL: return 4;
      case TETRAHEDRON: return 4;
      case PYRAMID: return 5;
      case PRISM: return 5;
      case HEXAHEDRON: return 6;
      default: SU2_MPI::Error("Invalid elementType.", CURRENT_FUNCTION); return 0;
    }
  }
```
Use naming conventions that more or less follow what is done.